examples/ota-provider-app/linux/main.cpp

```diff
@@ -306,9 +299,7 @@ OptionSet cmdLineOptions = { HandleOptions, cmdLineOptionsDef, "PROGRAM OPTIONS"
                              "  -x, --ignoreQueryImage <ignore count>\n"
                              "        The number of times to ignore the QueryImage Command and not send a response.\n"
                              "  -y, --ignoreApplyUpdate <ignore count>\n"
-                             "        The number of times to ignore the ApplyUpdateRequest Command and not send a response.\n"
-                             "  -P, --pollInterval <time in milliseconds>\n"
-                             "        Poll interval for the BDX transfer \n" };
+                             "        The number of times to ignore the ApplyUpdateRequest Command and not send a response.\n" };
```

Comment:
This same info is also in the README. Please also remove from there.

Reply:
I think this is for showing the options on the commandline for the user using the command